tessera.lic

```diff
@@ -0,0 +1,74 @@
+=begin
+  Documentation:  Tessera script based upon concept around Almanac.lic.
+=end
+
+custom_require.call(%w[common common-items drinfomon])
+
+class Tessera
+
+  def initialize
+    settings = get_settings
+    tessera_retry_interval = settings.tessera_retry_interval
+    startup_delay = settings.tessera_startup_delay
+
+    UserVars.tessera_last_use ||= Time.now - tessera_retry_interval
+    @no_use_scripts = settings.tessera_no_use_scripts 
+    @tessera = settings.tessera_noun
+
+    DRC.message("Tessera Startup Options:")
+    DRC.message("[tessera_noun: #{@tessera}]\n[tessera_startup_delay: #{startup_delay} (seconds)]\n[tessera_retry_interval: #{tessera_retry_interval} (seconds)]\n[tessera last attempt: #{UserVars.tessera_last_use}]\n[tessera_no_use_scripts: #{@no_use_scripts}]")
+
+    pause startup_delay
+    passive_loop
+  end
+
+  def use_tessera
+    settings = get_settings
+    tessera_retry_interval = settings.tessera_retry_interval
+
+    return if Time.now - UserVars.tessera_last_use < tessera_retry_interval
+    return if @no_use_scripts.any? { |name| Script.running?(name) }
+    return if XMLData.room_title.include? 'Carousel Chamber'
```

Comment:
What do you think about extracting this to a "no use rooms" setting like these?

https://github.com/rpherbig/dr-scripts/blob/567483a6bd76c4f8b82800f8b04fe846b63ebed0/profiles/base.yaml#L249

https://github.com/rpherbig/dr-scripts/blob/567483a6bd76c4f8b82800f8b04fe846b63ebed0/profiles/base.yaml#L1884